torch/testing/_internal/common_nn.py

```diff
@@ -4666,18 +4666,19 @@ def padding3d_circular(input, pad):
     ),
     dict(
         module_name='KLDivLoss',
-        input_fn=lambda: torch.rand(10, 10).log(),
+        input_fn=lambda: torch.rand(10, 10),
         target_fn=lambda: torch.rand(10, 10),
         reference_fn=lambda i, t, m:
             kldivloss_reference(i, t, get_reduction(m)),
         check_sum_reduction=True,
     ),
     dict(
         module_name='KLDivLoss',
-        input_fn=lambda: torch.rand(10, 10).log(),
-        target_fn=lambda: torch.rand(10, 10),
+        constructor=wraps(nn.KLDivLoss)(partial(nn.KLDivLoss, log_target=True)),
```

Comment:
This is an ugly hack, but I feel like this whole file is. Basically, this is a one-liner that is able to create `nn.KLDivLoss` with `log_target=True` and also have `.__name__ == "KLDivLoss"`. cc @thomasjpfan @jbschlosser